packages/fs-detectors/src/detect-builders.ts

```diff
@@ -453,9 +453,7 @@ function getApiMatches() {
 
   return [
     { src: 'middleware.[jt]s', use: `@vercel/node`, config },
-    { src: 'api/**/*.js', use: `@vercel/node`, config },
-    { src: 'api/**/*.mjs', use: `@vercel/node`, config },
-    { src: 'api/**/*.ts', use: `@vercel/node`, config },
+    { src: 'api/**/*.+(js|mjs|ts|tsx)', use: `@vercel/node`, config },
```

Comment:
It looks like `src` is a glob pattern. Is `+` supported in our system because it doesn't appear to be valid in bash:

<img width="335" alt="image" src="https://user-images.githubusercontent.com/97262/212377700-05cc6904-ed7e-4248-bcc8-d888f2699951.png">

```suggestion
    { src: 'api/**/*.(js|mjs|ts|tsx)', use: `@vercel/node`, config },
```

Reply:
@cb1kenobi As you can see the tests are passing. This is because we don't use bash, we use `minimatch`

https://github.com/isaacs/minimatch#usage

Reply:
Ah, it's an extglob. I saw the tests were passing, just didn't see it in the code. OK, cool, looks good!